proto/src/determined/api/v1/trial.proto

```diff
@@ -283,6 +283,18 @@ message GetExperimentTrialsResponse {
   Pagination pagination = 2;
 }
 
+// Get the remaining log retention days for a trial id.
+message GetTrialRemainingLogRetentionDaysRequest {
+  // The trial id.
+  int32 id = 1;
+}
+
+// Response to GetTrialRemainingLogRetentionDaysRequest.
+message GetTrialRemainingLogRetentionDaysResponse {
+  // The remaining log retention days for the trial id.
+  optional int32 remaining_days = 1;
```

Comment:
looks like API always return `remaining_days`, so can we remove `optional`, no?

Reply:
Based on the corner case explained before, we need to have the optional value